scenarios/sslstream.benchmarks.yml

```diff
@@ -0,0 +1,95 @@
+# Example:
+# crank --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/sslstream.benchmarks.yml --scenario read-write --profile aspnet-perf-lin
+
+imports:
+  - https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/aspnet.profiles.standard.yml
+
+variables:
+  # common parameters
+  serverPort: 5000
+  sendBufferSize: 32768 # bytes
+  receiveBufferSize: 32768 # bytes
+  tlsVersion: "1.3" # <1.2|1.3>

```

Comment:
should ve leave it to None as default? It may make it less predictable but it would track default use nicely IMHO

Reply:
It was default before, but @carnaviire argued that we should be explicit about the version for consistency.